packages/ndla-ui/src/MessageBox/MessageBox.tsx

```diff
@@ -157,7 +162,11 @@ export const MessageBox = ({ type, sticky = false, onClose, children, links, t }
           </IconWrapper>
           <TextWrapper>{children}</TextWrapper>
         </InfoWrapper>
-        {onClose && <CloseButton onClick={onCloseMessageBox} color="#444444" />}
+        {onClose && (
+          <CloseButtonWrapper>
+            <CloseButton onClick={onCloseMessageBox} />
+          </CloseButtonWrapper>
+        )}
```

Comment:
```suggestion
      {onClose ? <CloseButton type="button" onClick={onClose} /> : <CloseButton type="button" data-notion-close />}

```

Reply:
Noen av meldingsboksene skal ikke ha en lukkeknapp.

Reply:
Da er nok det tryggeste å legge til en ny prop `hideCloseButton`. Det er tilfeller hvor onClose ikke er definert, men hvor det brukes scripts for å legge til event-listener basert på `data-notion-close`.